src/pages/Inventory.page.js

```diff
@@ -12,4 +18,22 @@ export class InventoryPage extends BaseSwagLabPage {
     async addItemToCartById(id) {
         await this.addItemToCartBtns.nth(id).click();
     }
+
+    async performSortingList() {
+        await this.page.locator(this.productSortingList).click();
```

Comment:
Same here. In case we store `this.page.locator(this.productSortingList)`, no sence in having click method saved here.